demo-app/client/generated/controllers/DeleteOrder.ts

```diff
@@ -0,0 +1,34 @@
+/* tslint:disable:max-line-length */
```

Comment:
Should this folder be present at all? I would remove it from the repo.

Reply:
I believe new potential users might find it useful to see the generated files so that they can get a better sense of how the system works and how the generated files look like.

Reply:
Makes sense. Re-generation should be done in pre-publish stage, right?